_sass/modules/_feature-cards.scss

```diff
@@ -56,37 +41,9 @@
       height: $icon-size--m;
       margin: 20px 24px;
 
-      @media (max-width: $phone-medium) {
+      @media (max-width: $desktop-small) {
         margin: 16px 16px;
```

Comment:
You can use shorter variant  `margin: 16px;` if you wish.

Reply:
Done.